site/src/Librecores/ProjectRepoBundle/DataConfig.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Librecores\ProjectRepoBundle;
+/**
+ * This class holds the default data 
+ *
+ */
+
+class DataConfig
```

Comment:
This class is both named way too generic and in a place where typically no classes are in Symfony. Simply leave this data inside the migration, having it anywhere else doesn't help as a new migration needs to be created anyways if this data changes.